update readme, remove router binary, fix resolver fn (#172)
apollographql/odyssey-voyage-II-server: final/README.md

1. In a new terminal window, navigate to the `final` directory, then run `npm run launch`. This will run 4 local services, which you can learn about in the accompanying Odyssey course.

## Installing the router

> We have removed the router binary from this course, so when running the `final` code please re-download the router into the `router` directory.

Author: ```suggestion
When running the `final` code please be sure to download the router into the `router` directory.
```
